entrypoint.sh

```diff
@@ -31,7 +16,25 @@ if [[ ! -f "/root/.kube/config" ]]; then
 fi
 
 kube_client() {
-  kubectl "${INSECURE:+'--insecure-skip-tls-verify'}" --cluster "${CLUSTER}" --user "${USER}" --namespace "${NAMESPACE}" "${@}";
+  params=()
+
+  if [[ "$CLUSTER" ]]; then
```

Comment:
Doesn't this lack nonzero test `-n` ?

Reply:
According to Bash manual, `string` and `-n string` expressions are equivalent. 

```
CONDITIONAL EXPRESSIONS
 ...
       string
       -n string
              True if the length of string is non-zero.
```